src/platyPS/platyPS.psd1

```diff
@@ -18,19 +18,19 @@ ModuleVersion = '0.0.1'
 GUID = '0bdcabef-a4b7-4a6d-bf7e-d879817ebbff'
 
 # Author of this module
-Author = 'PowerShell team'
+Author = 'PowerShell'
```

Comment:
why are you keep insisting on changing fields that are not relevant for #406 ? :)
Can we have a single-line change for `PowerShellVersion = '3.0'` and open a separate PR for the rest, if you feel strongly about it?

Reply:
I agree with you in principle, but in this specific PR, this is a small change and already made.